app/rot-carrier/app.toml

```diff
@@ -195,12 +198,9 @@ task-slots = ["gpio_driver", "swd", "update_server"]
 name = "task-sp-measure"
 priority = 6
 max-sizes = {flash = 131072, ram = 8192}
```

Comment:
You almost certainly will be able to drop the flash size, I had to increase it for including part of the SP image to compare